Dockerfile

```diff
@@ -6,6 +6,8 @@ ENV NODE_ENV=production
 ENV ELASTIC_APM_JS_BASE_SERVICE_NAME=opbeans-react
 RUN npm install
 
+RUN npm install pm2 -g
+
 COPY --from=opbeans/opbeans-frontend:latest /app/ /app/client/
 
-CMD ["npm", "start"]
+CMD ["pm2-runtime"]
```

Comment:
I haven't tested it but I assume `pm2-runtime` works similar to `pm2` in that it will automatically pick up `ecosystem.config.js`

Reply:
I just made it more explicit.